.ci/azure-pipeline-azureml-notebook-test.yml

```diff
@@ -11,6 +11,11 @@ variables:
   value : 'eastus2'
 - name  : 'junitxml'
   value : 'reports/test-unit.xml'
+  
```

Comment:
So this .yml file does not set up any triggers, but you probably used the UI to set up nightly runs? Could you send me a link to the UI (via email) please - would like to understand how to do this myself in the future.